Dockerfile

```diff
@@ -1,5 +1,5 @@
-ARG NODE_VERSION=18
-FROM node:${NODE_VERSION}-alpine3.18
```

Comment:
The docker image were using is deprecated now. I think this is the cause of the font changes in all the snapshots?